<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckPwdDialog</class>
 <widget class="QDialog" name="CheckPwdDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>543</height>
    </rect>
   </property>
   <widget class="QWidget" name="containerWidget" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>420</width>
      <height>220</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI Light</family>
     </font>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>输入钱包密码：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="pwdLineEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="cancelBtn">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>140</y>
       <width>120</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI Light</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QToolButton" name="okBtn">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>120</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI Light</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QLabel" name="tipLabel2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>200</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="closeBtn">
     <property name="geometry">
      <rect>
       <x>395</x>
       <y>12</y>
       <width>13</width>
       <height>13</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
